src/app/components/TradeProfit/index.tsx

```diff
@@ -90,7 +66,7 @@ export const TradeProfit: React.FC<ITradeProfitProps> = ({
               <>
                 {t(translations.tradingHistoryPage.table.profitLabels.down)}
                 <span className="tw-text-warning">
-                  {toNumberFormat(Math.abs(profitDirection), 2)}
+                  {Number(realizedPnLPercent).toFixed(2)}
```

Comment:
Again, this should not come from the graph (I know this property exists on the graph, but it should be calculated the same way as it currently is from contract values)